### ISSUE TITLE
Plublish WindowTop to winget package manager tool

### ISSUE THREAD

**Issue by gileli121** [BiGilSoft/WindowTop#115]
This was also requested by users to be allowed to install WindowTop via Winget package manager

**Comment by vedantmgoyal2009** [comment]
@gileli121 since winget is now able to install apps from msstore, should it be removed from https://github.com/microsoft/winget-pkgs/tree/master/manifests/g/gileli121/windowtop? winget is now like a CLI for msstore, although there are some bugs which will be fixed soon.

**Comment by gileli121** [comment]
@vedantmgoyal2009 
Maybe it can be removed.
I am working on support to activate the ms store version with product key (in addition to the regular method that is purchasing it from ms store)

So both versions will be same anyway.

The question is if for some organisations Microsoft store is blocked by policy.

I saw one such case..

**Comment by vedantmgoyal2009** [comment]
Yeah, there're some orgs which block access to MS Store.

BTW, you can use WinGet Releaser (https://github.com/marketplace/actions/winget-releaser) to auto-update the app in winget-pkgs repository.

The docs are a bit outdated, so if you would like, I can create a PR to implement the workflow in your repository.

**Comment by gileli121** [comment]
@vedantmgoyal2009 
Sorry for the late reply.
Thank you, you can create PR.

**Comment by vedantmgoyal2009** [comment]
I saw your repository. Before implementing the workflow, you'll need to publish EXEs of the latest version outside the ZIP.

![image](https://user-images.githubusercontent.com/83997633/190551525-0daf7f31-8c38-4821-8094-9ecb9ad4d8f3.png)

**Comment by gileli121** [comment]
@vedantmgoyal2009 
Done:
https://github.com/BiGilSoft/WindowTop/releases/tag/v5.17.0

But I will keep publishing the installer in ZIP and link it to the description.
This is because it may help to prevent this smart screen prompt
![image](https://user-images.githubusercontent.com/17680514/190883269-c9b8c710-b9d4-462b-8d8e-d4c74937ed5f.png)

And I think that it may be better for the package manager to use the zip file for the same reason also (to help prevent this false positive smart screen prompt)

**Comment by gileli121** [comment]
Closed due to inactivity and low priority

**Comment by vedantmgoyal2009** [comment]
@gileli121 winget supports installers inside .zip archives now. maybe this can be opened?
